client/app/states/requests/details/details.state.js

```diff
@@ -1,21 +1,24 @@
 import templateUrl from './details.html';
 
 /** @ngInject */
-export function RequestsDetailsState(routerHelper) {
-  routerHelper.configureStates(getStates());
+export function RequestsDetailsState(routerHelper, RBAC) {
```

Comment:
aww this isn't a component yet :( was gonna say we should throw this in the component not the state, but i guess that'll be done at a later time